src/components/Organisms/AlertManager/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { useAlertHandlerContext } from "../../../contexts/alert_handler.jsx";
+import Alert from "../../Atoms/Alert/index.jsx";
+import React, { useState, useEffect } from "react";
+
+const AlertManager = () => {
+  const AlertHandlerContext = useAlertHandlerContext();
+
+  const setInterval = () => {
+    setTimeout(() => {
+      AlertHandlerContext.closeAlert();
+    }, 5000);
+  };
+  useEffect(() => {
+    if (AlertHandlerContext.visible == true) {
+      setInterval();
+    }
```

Comment:
```suggestion
    useEffect(() => {
      const setInterval = () => {
      setTimeout(() => {
        AlertHandlerContext.closeAlert();
      }, 5000);
    };
  
    if (AlertHandlerContext.visible == true) {
      setInterval();
    }
```
今回はシンプルなコンポーネントなので現状でも全く問題ありませんが、このようにuseEffectの中で関数を宣言することによってスコープが閉じ可読性が上がるというメリットが有ります。複雑なコンポーネントを作る際には意識してみてください！

スコープについて：https://jsprimer.net/basic/function-scope/